configure.ac

```diff
@@ -9,7 +9,7 @@ AC_PROG_CC
 AC_C_CONST
 AC_PROG_LIBTOOL
 
-AM_CONDITIONAL(GOT_PREFIX, test "x${prefix}" != "xNONE"])
+AM_CONDITIONAL(GOT_PREFIX, [test "x${prefix}" != "xNONE" && test "x${prefix}" != "x/usr"])
```

Comment:
GOT_PREFIX is not needed at all. My pull request #13 removes it. The problem with GOT_PREFIX is that it doesn't have a clear semantic - when it should be defined and what it should do. The implication is that if the user installs software to some location, includes and libraries already in that location should be used. The problem is that it's not always the case, and it's easier to add a directory to the includes than to remove it.